src/metabase/metabot.clj

```diff
@@ -235,7 +235,7 @@
    This will spin up a background thread that opens and maintains a Slack WebSocket connection."
   []
   (when (and (setting/get :slack-token)
```

Comment:
should this be `(slack-token)` as well? As mentioned in the docstring for `setting/get` it's an internal implementation detail and you shouldn't use it directly unless absolutely necessary